### PR TITLE
ref(ui) Replace icon-checkmark with SVG icons

### PR DESCRIPTION
**Pull request by markstory**
I also replaced other icons in the modules as opportunistic changes.

### CODE REVIEW

File: src/sentry/static/sentry/app/components/debugFileFeature.tsx [hunk 1]
```diff
@@ -31,9 +31,17 @@ const DebugFileFeature = ({available, feature}: Props) => {
...
+    icon = (
+      <IconWrapper>
+        <IconCheckmark size="sm" color="green500" />
+      </IconWrapper>
+    );
   } else if (available === false) {
-    icon = <Icon type="error" src="icon-close" />;
+    icon = (
+      <IconWrapper>
+        <IconClose size="sm" color="yellow500" />
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/debugFileFeature.tsx:42]: Is this intentional? error iconColor is red400

**Review by billyvg — APPROVED** [review]
✅

File: src/sentry/static/sentry/app/components/debugFileFeature.tsx [hunk 2]
```diff
@@ -31,9 +31,17 @@ const DebugFileFeature = ({available, feature}: Props) => {
   let icon: React.ReactNode = null;
 
   if (available === true) {
-    icon = <Icon type="success" src="icon-checkmark-sm" />;
+    icon = (
+      <IconWrapper>
+        <IconCheckmark size="sm" color="green500" />
```
Comment (matejminar) [inline src/sentry/static/sentry/app/components/debugFileFeature.tsx:36]: I believe that `size="sm"` is the default, so theoretically we don't have to pass that. (I like how explicit it is though).

**Review by matejminar — APPROVED** [review]

**Comment by priscilawebdev** [comment]
maybe it's because of the icon, but here it doesn't seem centralized to me :/

image

**Review by priscilawebdev — APPROVED** [review]

**Pull request merged**